js/components/operatorSignIn/attestation.tsx

```diff
@@ -47,22 +48,43 @@ export const Attestation = ({
     <div className="text-sm">
       Step 2 of 2
       <SignInText />
-      <SignaturePrompt defaultValue={defaultValue} onChange={setEntered} />
-      <SignatureHint badge={badge} signatureText={entered} />
-      <p className="my-3">
-        By pressing the button below I, <b className="fs-mask">{name}</b>,
-        confirm the above is true.
-      </p>
-      <button
-        className={className([
-          "block w-full md:max-w-64 mx-auto h-10 px-5 bg-gray-500 text-gray-200 rounded-md",
-          (!ready || loading) && "opacity-50",
-        ])}
-        onClick={onComplete}
-        disabled={!ready}
+      <form
+        onSubmit={(e) => {
+          e.preventDefault();
+        }}
       >
-        Complete Fit for Duty Check
-      </button>
+        <InputBox
+          title={"Operator Badge Number"}
+          defaultValue={defaultValue}
+          onChange={(evt) => {
+            setEnteredBadge(removeLeadingZero(evt.target.value));
+          }}
+        />
+        <SignatureHint badge={badge} signatureText={enteredBadge} />
+        <InputBox
+          title={"Radio Number"}
+          defaultValue={""}
+          onChange={(evt) => {
+            setEnteredRadio(evt.target.value);
+          }}
+        />
+        <p className="my-3">
+          By pressing the button below I, <b className="fs-mask">{name}</b>,
+          confirm the above is true.
+        </p>
+        <button
+          className={className([
+            "block w-full md:max-w-64 mx-auto h-10 px-5 bg-gray-500 text-gray-200 rounded-md",
+            (!valid || loading) && "opacity-50",
+          ])}
+          onClick={() => {
+            onComplete(enteredRadio);
```

Comment:
optional refactoring idea: I remember there was a thing about using form `onSubmit` instead of the button `onClick` for something else recently. Would it be better to make this button a submit button and do this call in the form's `onSubmit` handler instead too?